src/calculators/BarnesPointsCalc.ts

```diff
@@ -30,6 +30,7 @@ export type BarnesSimpleCategoryResults = {
 
 const PLACEHOLD_TEAM_NAME = 'Empty';
 const EXHIB_PENALTY_CODE = 'Exhib';
+const WOMENS_EVENT_REGEX_MATCHERS = [/WOMEN/, /GIRL/];
```

Comment:
FYI, you can add make a regex ignore case by appending i: /WOMEN/i, /GIRL/i.  This can be useful to avoid having to use tolowercase or touppercase when comparing.